packages/create-pantheon-decoupled-kit/src/templates/gatsby-wp/.env.example

```diff
@@ -1,5 +1,11 @@
-# Optional environment variables are commented out
+# The graphql endpoint of your WordPress instance
 WPGRAPHQL_URL=https://my-backend-site.pantheonsite.io/wp/graphql
-#PANTHEON_UPLOAD_PATH=/example
-#WP_APPLICATION_USERNAME=
-#WP_APPLICATION_PASSWORD=
+
+# Credentials for a valid user in the WordPress instance
+WP_APPLICATION_USERNAME=user
```

Comment:
Kinda thinking out loud here, but my concern is the username and password for the account. The application password is something different.

Would something like this make it better, or is it still confusing?

`# Username and Application Password for a valid user in the WordPress instance`

Reply:
Makes more sense to me, I forget this distinction myself quite often.